src/db/mongo/scripts/update-items.ts

```diff
@@ -0,0 +1,42 @@
+import { MONGO } from '../mongo';
+
+const updateInterval = 1000 * 60 * 60 * 12; // 12 hours
+const baseHeaders = {
+	Accept: 'application/json',
+	'User-Agent': 'SkyStats'
+};
+
+async function updateItems() {
+	try {
+		const timeNow = Date.now();
+		const response = await fetch('https://api.slothpixel.me/api/skyblock/items', {
+			headers: baseHeaders
+		});
+		const data = await response.json();
+
+		const items = Object.keys(data).map((skyblockId) => {
+			const skyblockItem = data[skyblockId];
+
+			const item = {
+				skyblock_id: skyblockId,
+				id: data.item_id,
+				damage: 0,
+				tier: 'common'
+			};
+
+			return Object.assign(item, skyblockItem);
+		});
+
+		await Promise.all(
```

Comment:
A bulk update call to mongo would be better here. This is called infrequently, but it's ~4.5k items.